app/src/full/java/io/homeassistant/companion/android/vehicle/HaCarAppService.kt

```diff
@@ -31,6 +34,8 @@ class HaCarAppService : CarAppService() {
 
     companion object {
         private const val TAG = "HaCarAppService"
+        var carInfo: CarInfo? = null
```

Comment:
This service should only be used by the Templates to display data and not for getting data; moreover this service will not be boun on Android Auto Project until the user taps on the application

Reply:
Personally speaking I am not a fan of the notification route but I understand its one of the only ways to go about it. I think a better approach would be to put things in the hand of the user to chose to send the notification or not. Ideally we can use `command_activity` or a new command to launch the app from the background when a certain command is received. Putting it in the hands of the user allows them to add their own conditions for when they want things to auto start.

Reply:
I tried to launch the activity in background using the same intent from the notification with something like this: 
```kotlin
val intent = Intent(Intent.ACTION_VIEW).setComponent(ComponentName(context, HaCarAppService::class.java))
CarPendingIntent.getCarApp(context, intent.hashCode(), intent, 0).send()
```
but it doesn't seem to work, I guess that you need to include it in a notification to get it working.

Reply:
to start the intent it should be something more like `context.startActivity(intent)` 

pending intent is actually reserved for things like the notification or for something else to start it.